docs/getting-started-with-scalardb.md

```diff
@@ -178,51 +178,61 @@ public class ElectronicMoneyWithTransaction extends ElectronicMoney {
     // Start a transaction
     DistributedTransaction tx = manager.start();
 
-    // Retrieve the current balance for id
-    Get get = new Get(new Key(ID, id));
-    Optional<Result> result = tx.get(get);
-
-    // Calculate the balance
-    int balance = amount;
-    if (result.isPresent()) {
-      int current = result.get().getValue(BALANCE).get().getAsInt();
-      balance += current;
+    try {
+      // Retrieve the current balance for id
+      Get get = new Get(new Key(ID, id));
+      Optional<Result> result = tx.get(get);
+
+      // Calculate the balance
+      int balance = amount;
+      if (result.isPresent()) {
+        int current = result.get().getValue(BALANCE).get().getAsInt();
+        balance += current;
+      }
+
+      // Update the balance
+      Put put = new Put(new Key(ID, id)).withValue(BALANCE, balance);
+      tx.put(put);
+
+      // Commit the transaction (records are automatically recovered in case of failure)
+      tx.commit();
+    } catch (Exception e) {
+      tx.abort();
+      throw e;
```

Comment:
We need catch `Exception` and abort the transaction here. This PR is just adding this thing.